Cargo.toml

```diff
@@ -9,7 +9,7 @@ homepage = "https://github.com/killercup/assert_cli"
 documentation = "http://docs.rs/assert_cli/"
 readme = "README.md"
 categories = ["development-tools::testing"]
-keywords = ["cli", "testing", "assert"]
+keywords = ["cli", "testing", "assert", "command", "line"]
```

Comment:
As two separate keywords? Okay.

Reply:
Maybe add the `command-line-interface` category as well?

Reply:
I'll give the latter a try and once its published, we'll see if crates.io handles it well.